client/modules/core/css/core.css

```diff
@@ -29,7 +29,8 @@ li.treeview > a > span::after {
   font-size: 10px;
   font-weight: 300;
   padding-right: 10px;
-  float: right;
+	position: absolute;
```

Comment:
Can you change the tab indentation to 2 spaces?  Thanks!

Reply:
No idea why it indented like that, but I’ll fix it up!